server/embed.go

```diff
@@ -2,17 +2,35 @@ package server
 
 import (
 	"fmt"
-	"github.com/bakape/meguca/common"
-	"net/http"
 	"strings"
+	"database/sql"
+	"net/http"
+
+	"github.com/bakape/meguca/common"
+	"github.com/bakape/meguca/db"
 
 	"github.com/badoux/goscraper"
-	"github.com/otium/ytdl"
+	"github.com/xenking/ytdl"
```

Comment:
Why did this change?

Reply:
ytdl isn't working at all right now, server just doesn't let the error message come through, API change or something.